packages/ui/src/components/Schemas/TextSchema.tsx

```diff
@@ -33,46 +34,85 @@ const TextSchemaUI = (
   }, [schema.data, schema.width, schema.fontName, schema.dynamicFontSize, schema.dynamicFontSize?.max, schema.dynamicFontSize?.min, schema.characterSpacing, font]);
 
   const style: React.CSSProperties = {
-    padding: 0,
-    resize: 'none',
     position: 'absolute',
-    fontFamily: schema.fontName ? `'${schema.fontName}'` : 'inherit',
+    top: 0,
+    padding: 0,
     height: schema.height * ZOOM,
     width: schema.width * ZOOM,
-    textAlign: schema.alignment ?? DEFAULT_ALIGNMENT,
+    resize: 'none',
+    fontFamily: schema.fontName ? `'${schema.fontName}'` : 'inherit',
+    color: schema.fontColor ? schema.fontColor : DEFAULT_FONT_COLOR,
     fontSize: `${dynamicFontSize ?? schema.fontSize ?? DEFAULT_FONT_SIZE}pt`,
     letterSpacing: `${schema.characterSpacing ?? DEFAULT_CHARACTER_SPACING}pt`,
     lineHeight: `${schema.lineHeight ?? DEFAULT_LINE_HEIGHT}em`,
+    textAlign: schema.alignment ?? DEFAULT_ALIGNMENT,
     whiteSpace: 'pre-line',
     wordBreak: 'break-word',
-    border: 'none',
-    color: schema.fontColor ? schema.fontColor : DEFAULT_FONT_COLOR,
     backgroundColor:
       schema.data && schema.backgroundColor ? schema.backgroundColor : 'rgb(242 244 255 / 75%)',
+    border: 'none',
   };
 
+  const schemaFontSize = dynamicFontSize ?? schema.fontSize ?? DEFAULT_FONT_SIZE;
```

Comment:
MEMO: Refactoring

Reply:
I prefer to extract the plain function and move hepler.ts.
and we can create `style` variable from`fontAlignmentValue` of the above function return value .